src/helpers.js

```diff
@@ -63,9 +64,8 @@ export async function sendJob(worker, config) {
 export function showError(givenMessage, givenDetail = null) {
   let detail
   let message
-  if (message instanceof Error) {
-    detail = message.stack
-    message = message.message
+  if (givenMessage instanceof Error) {
+    ({ stack: detail, message } = givenMessage)
```

Comment:
Since this syntax is uncommon, it might be worthwhile adding a comment explaining what it's doing and maybe even linking to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Assignment_without_declaration.

Reply:
I'd say the syntax is uncommon only because a lot of people don't know about it, I'll certainly be using it in a lot more places as the only reason I haven't been doing things like that is I didn't know the syntax.

So the question is: Comment it when the end result is obvious, or just start using it everywhere so people see it?

Reply:
I'm not so sure the end result is exactly obvious, and destructuring can be confusing if you're not used to it, with this form even more so.  Up to you though, I don't care very much.